src/controllers/itemdetailpage.js

```diff
@@ -1806,7 +1818,6 @@ define(["loading", "appRouter", "layoutManager", "connectionManager", "userSetti
             require(["chaptercardbuilder"], function (chaptercardbuilder) {
                 chaptercardbuilder.buildChapterCards(item, chapters, {
                     itemsContainer: scenesContent,
-                    width: 400,
```

Comment:
I cannot check chapters but I believe that 400px will still be here
https://github.com/jellyfin/jellyfin-web/blob/76b085982b2ca0183f946d1cd5b4c3f2d00cc57e/src/components/cardbuilder/chaptercardbuilder.js#L83